qrcode_generator.py

```diff
@@ -0,0 +1,12 @@
+import qrcode
+qr = qrcode.QRCode(
+    version=1,
+    error_correction=qrcode.constants.ERROR_CORRECT_L,
+    box_size=10,
+    border=4,
+)
+qr.add_data('Some data')
```

Comment:
You can pass this string via command line arguments like this `python3 qrcode_genrator.py "Some data"` and access it here via `sys.argv[1]`